solutions/product_solutions/NetInsight_Nimbra_Vision/01_netinsight-nimbra-vision-install.md

```diff
@@ -0,0 +1,68 @@
+
+> [!NOTE]
+> When you download the *NetInsight Nimbra Vision* package from the Catalog, the necessary connector is included and do not need to be installed separately.
+
+## Accessing the NetInsight Nimbra Vision solution
+
+To access the NetInsight Nimbra Vision application:
+
+1. Go to `http(s)://[DMA name]/root`.
+
+1. Select *NetInsight Nimbra Vision* to start using the application.
+
+![NetInsight Nimbra Vision](~/user-guide/images/netinsight-nimbra-vision-openapp.png)
+
+> [!NOTE]
+> This application enhances the functionality of existing NetInsight Nimbra Vision elements on your DMA. If these elements have not been configured yet, [add the necessary elements](#dataminer-element-configuration) using the [appropriate connectors](https://catalog.dataminer.services/details/e48af0b9-b52c-4106-b0e0-22c44ead85f5).
+
+## DataMiner element configuration
+
+The solution package installs the latest version of the NetInsight Nimbra Vision connector. You can also choose to install a specific version of the connector from the Catalog. However, note that the minimum required version of the connector is 1.0.1.1.
+
+When you have [created the elements](xref:Adding_elements) using this connector, specify the username and password on their Configuration page so requests can be authenticated. In addition, as this connector uses the HTTP protocol with [subscriptions](#subscriptions), you must **allow the Nimbra Vision platform to send asynchronous messages** to your DataMiner element.
+
+### Subscriptions configuration
+
+Nimbra Vision leverages webhooks and subscriptions to receive updates from specific endpoints with filtering capabilities. Currently, data filtering and new endpoints cannot be added manually or on demand. For support with additional endpoints or subscriptions, please contact Skyline Communications.
+
+To process asynchronous messages, you can use the default subscription address or define other entry points. The default address is `http://[DMS IP ADDRESS]/NimbraVision/subscriptions`.
+
+To define which endpoints to subscribe to, navigate to the ***Subscriptions** subpage and enable the desired endpoints.
+
+### Function Enablement
+
+The autoroute must be enabled for the provisioner to automatically route a path. If not, it will use whatever path the northbound application provided. If none was provided, then the network will route the channel but not create and book a path.
+
+To enable the above-mentioned functions:
+
+- Install the licenses on the page *Admin* > *System Administration* > *License Details*. In the *Action* menu, select *Install*.
+- Scheduling is enabled on the page *Admin* > *Circuit Provisioning* > *Scheduling Configurations*, by setting *Enabled* to *true*.
+- Manually trigger creation of resources using the node context menu by selecting *Scheduling* > *Refresh Resources*.
+- Default circuits to autoroute on page *Admin* > *Circuit Provisioning* > *Northbound Configuration*, setting *MSR Automatic Routing: true*
```

Comment:
Are these all steps that need to be followed in the indicated order? If yes, it would be better if we made this a numbered list instead of a bulleted list, and it would be better to rephrase all the steps as instructions in the same way.

Also, we should mention where the user can find these pages to prevent possible confusion.